src/components/MeetupList.tsx

```diff
@@ -115,24 +47,63 @@ const MeetupList: React.FC<IProps> = () => {
     setSearchField(event.target.value)
 
   return (
-    <div>
-      <StyledInput
+    <Box>
+      <Input
         onChange={handleSearch}
         placeholder={"Search by meetup title or location"}
+        display="block"
+        mr="auto"
+        ml="auto"
+        mb={6}
+        border="1px solid"
+        borderColor="searchBorder"
+        color="text"
+        p={2}
+        borderRadius={1}
+        w="100%"
+        _focus={{ outline: "primary solid 1px" }}
+        _placeholder={{ color: "text200" }}
       />
-      <Table>
+      <Box boxShadow="tableBoxShadow">
         {filteredMeetups.length ? (
           filteredMeetups.map((meetup, idx) => (
-            <Item key={idx} to={meetup.link}>
-              <LeftContainer>
-                <ItemNumber>{idx + 1}</ItemNumber>
-                <ItemTitle>{meetup.title}</ItemTitle>
-              </LeftContainer>
-              <RightContainer>
+            <Link
```

Comment:
Instead of wrapping this component with a Link, lets use the LinkOverlay component that Chakra offers for this specific scenario https://chakra-ui.com/docs/components/link-overlay/usage

I did a similar implementation in #7721